plugins/epos/plugin.py

```diff
@@ -1025,6 +1036,65 @@ def rda_a1_1_01d(self):
 
         return (points, msg_list)
 
+    def rda_a1_2_01d(self):
+        """Indicator RDA-A1_2-01D
+        This indicator is linked to the following principle: A1.2: The protocol allows for an
+        authentication and authorisation where necessary. More information about that principle
+        can be found here.
+        The indicator requires the way that access to the digital object can be authenticated and
+        authorised and that data accessibility is specifically described and adequately documented.
+        Technical proposal:
+
+        Returns
+        -------
+        points
+            A number between 0 and 100 to indicate how well this indicator is supported
+        msg
+            Message with the results or recommendations to improve this indicator
+        """
+        points = 0
+        msg = _(
+            "At the time, EPOS does not provide authentication or authorisation protocols"
+        )
+        if self.metadata_authentication:
+            points = 100
+            msg = "The authentication is given by: " + str(
+                self.metadata_authentication[0]
+            )
+        return points, msg
+
+    def rda_a2_01m(self):
+        """Indicator RDA-A1-01M
+        This indicator is linked to the following principle: A2: Metadata should be accessible even
+        when the data is no longer available. More information about that principle can be found
+        here.
+        The indicator intends to verify that information about a digital object is still available after
+        the object has been deleted or otherwise has been lost. If possible, the metadata that
+        remains available should also indicate why the object is no longer available.
+        Technical proposal:
+        Parameters
+        ----------
+        item_id : str
+            Digital Object identifier, which can be a generic one (DOI, PID), or an internal (e.g. an
+            identifier from the repo)
+        Returns
+        -------
+        points
+            A number between 0 and 100 to indicate how well this indicator is supported
+        msg
+            Message with the results or recommendations to improve this indicator
+        """
```

Comment:
Use latest docstring format

Reply:
Updated

Reply:
Same comment as above regarding the description following the old format

Reply:
Changed